nes/config.py

```
@@ -36,44 +32,54 @@ def get_db_path(cls, override_path: Optional[str] = None) -> Path:
+                "  file+memcached:///absolute/path/to/nes-db/v2"
+            )
+
+        parsed = urlparse(database_url)
+        # Support both file:// and file+memcached:// protocols
+        if parsed.scheme not in ("file", "file+memcached"):
+            raise ValueError(
+                f"NES_DB_URL must use 'file://' or 'file+memcached://' protocol, got '{parsed.scheme}://'. "
+                f"Examples:\n"
+                f"  file:///absolute/path/to/nes-db/v2\n"
+                f"  file+memcached:///absolute/path/to/nes-db/v2"
+            )
```

Ruff (0.15.4)

[warning] Avoid specifying long messages outside the exception class

(TRY003)